### PR TITLE
Embedcolor on market and auction infos

### PR DESCRIPTION
**Pull request by WitherredAway**
Would this work for the FR #261 ?

### CODE REVIEW

File: cogs/auctions.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ async def end_auction(self, auction):
                 print("Error trading auction logs.")
 
     def make_base_embed(self, author, pokemon, auction_id):
-        embed = self.bot.Embed()
+        embed = self.bot.Embed(color=pokemon.color or helpers.constants.PINK)
```
Comment (oliver-ni) [inline cogs/auctions.py:129]:
```suggestion
        embed = self.bot.Embed(color=pokemon.color or constants.PINK)
```

File: cogs/market.py [hunk 2]
```diff
@@ -346,7 +346,7 @@ async def info(self, ctx, id: int):
 
         pokemon = self.bot.mongo.EmbeddedPokemon.build_from_mongo(listing["pokemon"])
 
-        embed = self.bot.Embed()
+        embed = self.bot.Embed(color=pokemon.color or helpers.constants.PINK)
```
Comment (oliver-ni) [inline cogs/market.py:349]:
```suggestion
        embed = self.bot.Embed(color=pokemon.color or constants.PINK)
```

**Review by oliver-ni — CHANGES_REQUESTED** [review]

**Comment by WitherredAway** [comment]
Added requested fixes

**Review by oliver-ni — APPROVED** [review]
lgtm

**Pull request merged**